src/parser.rs

```diff
@@ -2921,9 +2921,8 @@ impl<'a> Parser<'a> {
         } else if self.parse_keywords(&[Keyword::ON, Keyword::UPDATE])
             && dialect_of!(self is MySqlDialect)
```

Comment:
@CEOJINSUNG please add the `GenericDialect` here.

Reply:
Thank you for advising good comment